src/components/OSCALImplementedComponents.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { styled } from "@mui/material/styles";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableRow from "@mui/material/TableRow";
+
+const ImplementedComponentsTable = styled(TableContainer)(
+  ({ theme }) => `
+  margin-top: ${theme.spacing(2)};
+  display: flex;
+  flex-direction: column;
+  text-transform: capitalize;
+`
+);
+
+export default function OSCALImplementedComponents(props) {
+  const getComponent = (compUuid) =>
+    props.components?.find((component) => component.uuid === compUuid);
+
+  return (
+    <ImplementedComponentsTable>
+      <Table size="small">
+        <TableBody>
+          {props.implementedComponents &&
+            Object.entries(props.implementedComponents).map(([key, comp]) => (
```

Comment:
I think you should be able to map `implementedComponents` without using `Objects.entires`. It should be an array. Same goes for a lot of places we are doing `Object.entries`.

Reply:
So just something like this:

```suggestion
            props.implementedComponents.map(([key, comp]) => (
```

Reply:
Most objects have a `uuid`. You should not need to use `key` I do not think. You should be able to just use `comp.uuid` as a key. Keep in mind that `comp.uuid` is an object, so you will need to use it as a `String` for the key.

Reply:
```suggestion
            props.implementedComponents.map((comp) => (
```